content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,105 @@
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, a new major version of our free open source DJ software, [available right now](https://mixxx.org/download/). It's been two years since the last release due to the large number of great new features and important changes under the hood.
+
+The 2.4 release includes the following highlights:
+
+### EngineOS export
+
+Mixxx 2.4 adds support for exporting crates, playlists and the library to Engine DJ OS devices, including Denon and Numark standalone controllers.
+@Video(https://www.youtube.com/watch?v=Qle9Y1KdD2Y)
+This allows users to prepare their tracks on a laptop using Mixxx and export them to a USB stick for use on a standalone controller at their gig.
+
+### Saved Loops
+
+Mixxx 2.4 adds support for Saved Loops, which can be assigned to any Hot Cue slot.
+If you already have tracks from Serato DJ with Saved Loops, Mixxx 2.4 will import them automatically.
+
+@Video(https://www.youtube.com/watch?v=f8hI_uv9zHY)
+
+### Effect Chains
+
+Mixxx 2.4 introduces Effect Chains, a set of effects which can be saved and reloaded with all their parameters and individual Meta Knob mappings.
+Effect Chains can also be exported and imported, which allows you to share them with other Mixxx DJs.
+Once you've created an Effect Chain, you can create a custom Super Knob mapping which turns the chain into a Quick-Effect, where you take creative control with a single knob.
+Effect parameters can be reordered and or hidden.
+In the effect units, Mixxx 2.4 also displays the value and unit of effect knob parameters while they are being adjusted.
+
+### Track Menu Enhancements
+
+The track menu in Mixxx 2.4 is enhanced by several new features:
+
+- Select loaded track in library.
+- Analyze/re-analyze with variable/constant BPM on a per-track basis.
+- Look up track metadata at Discogs, SoundCloud and LastFM.
+- Remove track files from disk.
+
+### Sync Lock Improvements
+
+Mixxx 2.4 adds support for manually selecting the Sync Leader by clicking on the new button with the crown symbol next to the Sync button.
+This mode is useful when tracks have variable BPM.
+All decks with sync enabled will follow the changing BPM of the Sync Leader.
+@Video(https://www.youtube.com/watch?v=REiG8DBKr-M)
+
+### Many other changes
+
+##### Soundstretcher / Key-Shift Algorithms
+
+Mixxx 2.4 now supports three different key shift algorithms:
+
+- Soundtouch (Basic audio quality - medium CPU usage)
+- Rubberband R2 (Good audio quality - higher CPU usage)
+- Rubberband R3 (High audio quality - very high CPU usage)
+
+##### Major rework of the Waveform Rendering code
+
+The waveform display has been rewritten, solving some longstanding performance issues, especially on macOS.
+
+##### Native macOS ARM builds for Apple silicon
+
+##### Controller Backend
+
+Modernized Javascript ES7 based controller mapping system.
+HID backend rework.
+
+## Mixxx 2.4 Factsheet
+
+##### Supported platforms:
```

Comment:
```suggestion
##### Supported Platforms
```

Reply:
I just noticed: the headline levels don't match up. The parent section uses `##` so this one should use `###` (this does not videolate the spec but is discouraged: https://html.spec.whatwg.org/multipage/sections.html#headings-and-outlines). 

Is the font too big with `###`? If so, we should fix it in the CSS file.

Reply:
Agree -- I'll correct the markdown.

Btw we can also force some header levels to Title Case so that happens automatically as well